scalableminds/webknossos
# Add owner information to organization

- Added owner information to organization REST API route and organization page. 
  - Flo and I decided that we did not want share the whole orga owner user object through the API but rather just a string since some API calls expose the default orga without any authentication
- Also added the owner name to some of the pricing tooltips to help guide people to right person. 
- Added a button to the tooltip/popover in case you are the owner to upgrade seamlessly.

### Steps to test:
- Hover over any feature that is restricted to certain plans
- Check out the owner name in the organization page

### Issues:
- contributes to #6509 

------
(Please delete unneeded items, merge only when none are left open)
